test/LocalNeighborhood/test_ln.py

```diff
@@ -8,10 +8,6 @@
 
 config.init_from_file("config/config.yaml")
 
-# TODO consider refactoring to simplify the import
-# Modify the path because of the - in the directory
-SPRAS_ROOT = Path(__file__).parent.parent.parent.absolute()
-sys.path.append(str(Path(SPRAS_ROOT, 'docker-wrappers', 'LocalNeighborhood')))
 from local_neighborhood import local_neighborhood
```

Comment:
This should import the LocalNeighborhood algorithm instead (otherwise, your tests fail here.)